tests/server.spec.js

```diff
@@ -0,0 +1,48 @@
+// import DB from "../database/index";
+// import server from "../server/index";
+// import client from "../client/source/index";
+var req = require("supertest")
+
+
+describe('API communication: ', () => {
+  var app = req('http://localhost:3007');
+
+  describe('GET requests to /updates/:projectId', () => {
+
+    test('The database should get seeded with 100 entries', () => {
+      app.get('/updates/100')
+        .then(response => {
+          expect(response.status).toBe(200);
+        })
+    });
+
+    test('Returns a 200 response for valid projectIDs', () => {
```

Comment:
This is pretty much the same test as above. I would either refactor the test above or delete this one if you're going to keep it